src/containers/Network/Validators.tsx

```diff
@@ -23,41 +25,50 @@ export const Validators = () => {
     refetchOnMount: true,
   })
 
-  const mergeLatest = (validators: any = {}, live: any = {}) => {
-    const updated: any = {}
+  function mergeLatest(
+    validators: Record<string, ValidatorResponse>,
+    live: Record<string, StreamValidator>,
+  ): Record<string, StreamValidator> {
+    const updated: Record<string, StreamValidator> = {}
     const keys = new Set(Object.keys(validators).concat(Object.keys(live)))
     keys.forEach((d: string) => {
-      updated[d] = validators[d] || live[d]
-      if (live[d] && live[d].ledger_index > updated[d].ledger_index) {
-        updated[d].ledger_index = live[d].ledger_index
-        updated[d].ledger_hash = live[d].ledger_hash
+      const newData: StreamValidator = validators[d] || live[d]
+      if (newData.ledger_index == null && live[d] && live[d].ledger_index) {
+        // VHS uses `current_index` instead of `ledger_index`
```

Comment:
The VHS data is updated slower than the `Streams` data. So `ledger_index` might jump couple of index instead of increasing one by one as of now. Is that the behavior we want?

Reply:
The VHS data should never override the Streams data if the Streams data exists - which I'm pretty sure is what happens here.

Reply:
Yes you're right. I got confused between validators and live.